Gemfile

```diff
@@ -38,11 +38,15 @@ gem 'googlecharts', '~> 1.6.12'
 gem 'tweetstream', git: 'https://github.com/GSA/tweetstream'
 gem 'twitter', git: 'https://github.com/GSA/twitter.git', branch: '5-stable'
 gem 'flickraw', '~> 0.9.9'
-gem 'active_scaffold', '~> 3.6.2'
+gem 'active_scaffold', '~> 3.6.0'
```

Comment:
Yes, I had to downgrade `active_scaffold`. This is the result of them changing the rails requirements from `rails (>= 4.2.0)` to `rails (>= 4.2.0, < 6.2)` in 3.6.1:
https://github.com/activescaffold/active_scaffold/commit/c89a670462f3da490177ec28667af3c7455de344#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fR13

As tests are passing, and my local smoke-tests of super admin pass, I'm fairly comfortable with the downgrade. It's possible that a full test on staging will surface problems, but we can tackle those at that point.